package.json

```diff
@@ -13,11 +13,11 @@
     "lint": "./node_modules/eslint/bin/eslint.js 'src/**/*.{js,ts}' --max-warnings 0"
   },
   "engines": {
-    "node": "16"
+    "node": "^16.19.1 || ^18.17.1"
```

Comment:
why are we keeping 16 as an option here?

Reply:
Because this is included in the Hypersync SDK and there's no reason to force our SDK customers (few as they are) to upgrade to 18.